src/test/resources/application.properties

```diff
@@ -33,3 +33,5 @@ processor.email.bop_apitoken=addme
 processor.email.bop_client_id=policies
 processor.email.bop_env=qa
 processor.email.no_reply=no-reply@redhat.com
+
+quarkus.scheduler.enabled=false
```

Comment:
Shouldn't that be enabled?

Reply:
This is a test setting, I think is fine disabled.